extractor/src/test/suite/scala/com/digitalasset/extractor/TlsSpec.scala

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.extractor
+
+import java.io.File
+import java.nio.file.{Files, Path}
+
+import com.digitalasset.daml.bazeltools.BazelRunfiles._
+import com.digitalasset.extractor.config.SnapshotEndSetting
+import com.digitalasset.extractor.services.ExtractorFixture
+import com.digitalasset.extractor.targets.TextPrintTarget
+import com.digitalasset.ledger.api.testing.utils.SuiteResourceManagementAroundAll
+import com.digitalasset.ledger.api.tls.TlsConfiguration
+import com.digitalasset.platform.sandbox.config.SandboxConfig
+import org.apache.commons.io.FileUtils
+import org.scalatest._
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
+class TlsSpec
+    extends FlatSpec
+    with Suite
+    with SuiteResourceManagementAroundAll
+    with ExtractorFixture
+    with Matchers {
+
+  override protected def darFile = new File(rlocation("extractor/VeryLargeArchive.dar"))
+
+  private def extractCerts: Path = {
```

Comment:
There's side effects here, so I'd appreciate a `()`.

Reply:
I removed the entire `extractCerts` method. (All this code was copied from another spec in the sandbox tests). Should be cleaner now.